org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoPanel.java

```diff
@@ -1353,7 +1357,8 @@ protected String getUserOrderClause() {
 	 * @return
 	 */
 	protected String getUserOrderClause(int col) {
-		String colsql = p_layout[col].getColSQL().trim();
```

Comment:
should always reset joinTable to empty here.

Reply:
done

Reply:
not seeing that changes, I means we should reset `joinTableForUserOrder = null` here

Reply:
In the latest commit I moved it to InfoWindow:2145.

I've found issues with Trl tables and using SELECT DISTINCT with ORDER BY - committing fixes soon..